src/encoded/static/components/static-pages/components/StatisticsPageViewBody.js

```diff
@@ -0,0 +1,1092 @@
+            expsets_released_vs_internal, /* expsets_created, */ chartToggles, smoothEdges, width, onChartToggle, onSmoothEdgeToggle
+        } = this.props;
+
+        if (!mounted || (!expsets_released)){
+            return <div className="stats-charts-container" key="charts" id="submissions"><LoadingIcon/></div>;
+        }
+
+        if (loadingStatus === 'failed'){
+            return <div className="stats-charts-container" key="charts" id="submissions"><ErrorIcon/></div>;
+        }
+
+        const anyExpandedCharts = _.any(_.values(chartToggles));
+        const commonContainerProps = {
+            'onToggle' : onChartToggle, chartToggles, windowWidth,
+            'defaultColSize' : '12', 'defaultHeight' : anyExpandedCharts ? 200 : 250
+        };
+        const showInternalReleaseCharts = session && expsets_released_internal && expsets_released_vs_internal;
+        const commonChartProps = { 'curveFxn' : smoothEdges ? d3.curveMonotoneX : d3.curveStepAfter };
+
+        return (
+            <div className="stats-charts-container" key="charts" id="submissions">
+
+                { showInternalReleaseCharts ?
+
+                    <ColorScaleProvider width={width} colorScale={SubmissionsStatsView.colorScaleForPublicVsInternal}>
+
+                        <AreaChartContainer {...commonContainerProps} id="expsets_released_vs_internal"
+                            title={<h4 className="text-300 mt-0"><span className="text-500">Experiment Sets</span> - internal vs public release</h4>}>
+                            <AreaChart {...commonChartProps} data={expsets_released_vs_internal} />
+                        </AreaChartContainer>
+
+                        <hr/>
+
+                    </ColorScaleProvider>
+
+                    : null }
+
+                <ColorScaleProvider width={width} resetScalesWhenChange={expsets_released}>
+
+                    <GroupByDropdown {...{ currentGroupBy, groupByOptions, handleGroupByChange, loadingStatus }} title="Group Charts Below By">
+                        <div className="inline-block ml-15">
+                            <Checkbox checked={smoothEdges} onChange={onSmoothEdgeToggle}>Smooth Edges</Checkbox>
+                        </div>
+                    </GroupByDropdown>
+
+                    <hr/>
+
+                    <HorizontalD3ScaleLegend {...{ loadingStatus }} />
+
+                    <AreaChartContainer {...commonContainerProps} id="expsets_released" title={
+                        <h4 className="text-300 mt-0">
+                            <span className="text-500">Experiment Sets</span> - { session ? 'publicly released' : 'released' }
+                        </h4>
+                    }>
+                        <AreaChart {...commonChartProps} data={expsets_released} />
+                    </AreaChartContainer>
+
+                    {/* expsets_created ?           // ~=== 'Experiment Sets Submitted'
```

Comment:
You've got commented out JSX in multiple areas, but the purpose in keeping it around is not clear.

Reply:
Charts that we had before but then disabled for some reason or another (I forgot). I guess we could remove from repo completely, but there's chance we might want to bring back.

Reply:
True, but unless it's immediately clear to what these are a comment saying so would be nice.